build/benchmarks-ci-01.yml

```diff
@@ -269,7 +269,7 @@ jobs:
   - template: nativeaot-scenarios.yml
     parameters:
       connection: ASP.NET Benchmarks Queue 3
-      arguments: "$(ciProfile) --profile intel-win-app --profile intel-lin-load "
+      arguments: "$(ciProfile) --profile intel-win-app --profile intel-lin-load --profile amd-lin2-db "
```

Comment:
@sebastienros - one inconsistency I noticed was that 

* `Intel Windows` leg uses: `--profile intel-lin-load`
* `Intel Linux` leg uses: `--profile amd-lin2-load`

What are the implications of using a different load profile? For the `Fortunes` benchmark, I see we are using the same load profile. Literally everything in the command is different besides 

![image](https://user-images.githubusercontent.com/8291187/233137104-5bd6923a-d674-4b85-b9a4-7d61c499d98a.png)

Reply:
Not sure, I'll leave to @sebastienros to reply to this PR.

Reply:
@sebastienros explained it to me offline. this is fine because the load isn't the bottleneck on the app.